src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -13978,6 +13978,86 @@ public class QueryAsserter<TContext> : QueryAsserterBase
             }
         }
 
+        [Fact]
+        [WorkItem(38403, "https://github.com/dotnet/roslyn/issues/38403")]
+        public void Overriding_36()
+        {
+            var source =
+@"
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
Consider removing these `using` statements from the two tests since they only add diagnostics.